elements/noflo-component-editor.html

```diff
@@ -106,6 +106,35 @@
   return c;
 };
     </script>
+    <script type="text/plain" id="ES6Example">
```

Comment:
The convention is to use `<script type="text/babel"` or `<script type="text/ecmascript-6"`
Cf. https://babeljs.io/docs/usage/browser/